code/__DEFINES/spawner/_spawner.dm

```diff
@@ -29,34 +29,56 @@
 
 #define LOOT_LEVEL_VERY_HIG 20000
 
+// FACTION KEYWORDS
+#define SPAWN_ASTERS "asters"
+#define SPAWN_EXCELSIOR "excelsior"
+#define SPAWN_FROZEN_STAR "frozen_star"
+#define SPAWN_IRONHAMMER "ironhammer"
+#define SPAWN_NANOTRASEN "nanotrasen"
+#define SPAWN_NEOTHEOLOGY "neotheology"
+#define SPAWN_MOEBIUS "moebius"
+#define SPAWN_SERBIAN "serbian"
+#define SPAWN_ONESTAR "onestar"
+#define SPAWN_TECHNOMANCER "technomancer"
+
+#define SPAWN_CLOWN "clown"
+
 //	Items
 #define SPAWN_OBJ "obj"
 #define SPAWN_ITEM "item"
 #define SPAWN_WEAPON "weapon"
 #define SPAWN_DICE "dice"
 
-#define SPAWN_TAG_ITEM		MERGE_SPAWN_TAGS(SPAWN_OBJ, SPAWN_ITEM)
-#define SPAWN_TAG_WEAPON	MERGE_SPAWN_TAGS(SPAWN_TAG_ITEM, SPAWN_WEAPON)
+#define SPAWN_TAG_ITEM MERGE_SPAWN_TAGS(SPAWN_OBJ, SPAWN_ITEM)
+#define SPAWN_TAG_WEAPON MERGE_SPAWN_TAGS(SPAWN_TAG_ITEM, SPAWN_WEAPON)
 
-#define SPAWN_TAG_DICE				MERGE_SPAWN_TAGS(SPAWN_TAG_ITEM, SPAWN_DICE)
-#define SPAWN_TAG_ITEM_BOTANICAL	MERGE_SPAWN_TAGS(SPAWN_TAG_ITEM, "botanical")
-#define SPAWN_TAG_ITEM_CLOWN		MERGE_SPAWN_TAGS(SPAWN_TAG_ITEM, SPAWN_CLOWN)
+#define SPAWN_TAG_DICE MERGE_SPAWN_TAGS(SPAWN_TAG_ITEM, SPAWN_DICE)
+#define SPAWN_TAG_ITEM_BOTANICAL MERGE_SPAWN_TAGS(SPAWN_TAG_ITEM, "botanical")
+#define SPAWN_TAG_ITEM_CLOWN MERGE_SPAWN_TAGS(SPAWN_TAG_ITEM, SPAWN_CLOWN)
 
 
 #define SPAWN_RARE_ITEM "rare_item"
 
-#define SPAWN_TAG_RARE_ITEM			MERGE_SPAWN_TAGS(SPAWN_TAG_ITEM, SPAWN_RARE_ITEM)
+#define SPAWN_TAG_RARE_ITEM	MERGE_SPAWN_TAGS(SPAWN_TAG_ITEM, SPAWN_RARE_ITEM)
+
+// DEPARTMENT ROLES
+#define SPAWN_SCIENCE "science"
+#define SPAWN_MEDICAL "medical"
+
+#define SPAWN_TAG_MEDICAL MERGE_SPAWN_TAGS(SPAWN_TAG_ITEM, SPAWN_MEDICAL)
+#define SPAWN_TAG_SCIENCE MERGE_SPAWN_TAGS(SPAWN_TAG_ITEM, SPAWN_SCIENCE)
+#define SPAWN_TAG_SCIENCE_JUNK MERGE_SPAWN_TAGS(SPAWN_TAG_SCIENCE, SPAWN_JUNK)
```

Comment:
I think this would make this tag spawn both junk and science items, not items that are both junk and science

Reply:
It would make the tagged item both science and junk. Spawners looking for SPAWN_JUNK or SPAWN_SCIENCE would spawn this. They shouldn't be using SPAWN_TAG_WHATEVER to spawn things, anyway.